app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -507,299 +535,309 @@ function Form(
         }
       `}</style>
       <div className="flex overflow-y-auto scrollable-container pt-1">
-        <form
-          id={FORM_ID}
-          onSubmit={onFormSubmit}
-          {...others}
-          className="flex sm:w-1/2 flex-col"
-        >
-          <div className="flex flex-col">
-            <div className="flex flex-row items-end mb-3">
-              <FormLabel
-                label="Subject"
-                htmlFor="subject"
-                className="flex-1"
-                copyButton={{ text: subject, label: 'Copy subject' }}
-                extraLabel={
-                  <SubjectSwitchButton
-                    subject={subject}
-                    setSubject={setSubject}
+        <div className="flex sm:w-1/2 flex-col">
```

Comment:
The diff below looks scary because of this root level wrapper element change. Almost nothing changed below except for a level of indentation and positioning in line.

Reply:
Maybe not would be a good time to actually re-factor this huge component into smaller ones ?

Reply:
yeah it just keeps growing but honestly, this is like the most used and impactful piece of code in the whole app so kinda nervous to do that along with a PR. let me do a separate PR for it.